NWEEI/NWEEI/Models/Organization.cs

```diff
@@ -7,19 +7,25 @@ namespace NWEEI.Models
 {
     public class Organization
     {
-        private List<Tag> tags = new List<Tag>();
+        //private List<Tag> tags = new List<Tag>();
 
         public int OrganizationID { get; set; }
 
         [Required(ErrorMessage = "Name is required.")]
         public string Name { get; set; }
+
+        [Required(ErrorMessage = "ShortName is required.")]
```

Comment:
Smart. I was thinking of producing an acronym dynamically, but this is a much better option. Maybe we should add a max length as well?